appium/webdriver/extensions/action_helpers.py

```diff
@@ -149,3 +149,87 @@ def flick(self, start_x, start_y, end_x, end_y):
             .release()
         action.perform()
         return self
+
+    def zoom_element(self, element):
```

Comment:
Could you follow the same format to existing docstring? Thanks.

[references]
https://github.com/appium/python-client/blob/57fafcb4c32a38d170fc8658fe017fce19d2ee44/appium/webdriver/webelement.py#L77-L88
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
@ki4070ma Sure.